### PR TITLE
Add more staff logs

### PR DESCRIPTION
**Pull request by DatBoiWeep**

### CODE REVIEW

File: resources/erp_adminmenu/client/client.lua [hunk 1]
```diff
@@ -187,7 +191,15 @@ CreateThread(function()
       SetEntityVisible(yayeetEntity, false, false)
       SetLocalPlayerVisibleLocally(true)
       SetEntityAlpha(yayeetEntity, 50, false)
+      if not alert then
+        NotifyStaff(PlayerId(), 'has started using the cloak command.')
+        alert = true
+      end
     else
+      if alert and not Cloack then
```
Comment (minipunch) [inline resources/erp_adminmenu/client/client.lua:199]:
```suggestion
      if alert and not Cloak then
```

File: resources/erp_adminmenu/client/client.lua [hunk 2]
```diff
@@ -187,7 +191,15 @@ CreateThread(function()
       SetEntityVisible(yayeetEntity, false, false)
       SetLocalPlayerVisibleLocally(true)
       SetEntityAlpha(yayeetEntity, 50, false)
+      if not alert then
```
Comment (minipunch) [inline resources/erp_adminmenu/client/client.lua:194]:
```suggestion
      if not alerted then
```

File: resources/erp_adminmenu/server/server.lua [hunk 3]
```diff
@@ -498,4 +511,10 @@ RegisterCommand("closestveh", function(source, args, rawCommand)
 	end
 
 	print("Does Entity Exist: "..DoesEntityExist(info.obj).."\nOwner: "..NetworkGetEntityOwner(info.obj).."\nDistance: "..info.dist)
-end, false) -- set this to false to allow anyone.
\ No newline at end of file
+end, false) -- set this to false to allow anyone.
+
+function LogAndNotify(source, text)
+	Wait(150) -- To avoid display in text-chat
```
Comment (minipunch) [inline resources/erp_adminmenu/server/server.lua:517]: ???

File: resources/erp_adminmenu/client/client.lua [hunk 4]
```diff
@@ -58,6 +58,7 @@ end)
 
 RegisterNUICallback('noclip', function(data, cb)
   TriggerServerEvent('erp_adminmenu:toggleyayeet')
+  -- NotifyStaff(PlayerId(), 'has started/stopped NoClip command.')
```
Comment (minipunch) [inline resources/erp_adminmenu/client/client.lua:61]:
```suggestion
```

Can u plz remove these comments if not useful

**Review by minipunch — CHANGES_REQUESTED** [review]
see comments

File: resources/erp_adminmenu/server/server.lua (hunk 3 above)
Comment (DatBoiWeep) [reply resources/erp_adminmenu/server/server.lua:517]: If I take it off. It shows up on the text chat like this 💀 

image

Comment (minipunch) [reply resources/erp_adminmenu/server/server.lua:517]: You mean the coloring? You can fix that by prefixing the string with `^0` to make it white for example

**Review by minipunch — APPROVED** [review]

**Pull request merged**